docker-compose-sunbird/docker-compose.yml

```diff
@@ -95,6 +96,7 @@ services:
       - CREDENTIAL_SERVICE_BASE_URL=${CREDENTIAL_SERVICE_BASE_URL}
       - JWKS_URI=${JWKS_URI}
       - ENABLE_AUTH=${ENABLE_AUTH}
+      - QR_TYPE=W3C_VC
```

Comment:
What exactly is this environment variable is for ?

Reply:
this is required variable for sunbird new image

Reply:
ok